var/spack/repos/builtin/packages/comgr/package.py

```diff
@@ -118,6 +120,8 @@ def url_for_version(self, version):
         "6.3.0",
         "6.3.1",
         "6.3.2",
+        "6.3.2",
```

Comment:
Need to remove this 6.3.2

Reply:
Good catch. Fixed it just now. Thank you.